ServerCore/Pages/EventSpecific/PH20/ThePlayThatGoesWrong.cshtml

```diff
@@ -0,0 +1,3684 @@
+@page "/{eventId}/{puzzleId}/ThePlayThatGoesWrong.html"
+@model ServerCore.Pages.EventSpecific.PH20.ThePlayThatGoesWrongModel
+
+@{
+  Layout = null;
```

Comment:
So you intend for this page to have n header bar, etc.? Why?

Reply:
I don't want to use the default layout for two reasons:  (1) there's no way for me to add material to the <head> element, and (2) it confines the body to a pretty narrow window and I want the whole width of the browser.  It might be nice to have the header and footer, once they're settled on.